LayoutTests/imported/w3c/web-platform-tests/webrtc-encoded-transform/RTCRtpScriptTransform-encoded-transform-drop-frames-worker.js

```diff
@@ -0,0 +1,34 @@
+let frameRtpTimestamp;
+
+// Dropping all but the fifth frame on the sender,
+// verifying the receiver only gets that frame.
+onrtctransform = (event) => {
+    const transformer = event.transformer;
+
+    transformer.reader = transformer.readable.getReader();
+    transformer.writer = transformer.writable.getWriter();
+    let countRead = 0;
+
+    async function process(transformer)
+    {
+        const chunk = await transformer.reader.read();
+        countRead++;
+        if (chunk.done)
+            return;
+        if (transformer.options.name === 'sender') {
+            if(countRead == 5) {
```

Comment:
Tiny nit: should have a space between `if` and `(`

Reply:
Given this is a WPT test, it seems best not to change it here as import time but in a follow-up